crypto/kyber/p_kyber.c

```diff
@@ -29,6 +29,53 @@ static int pkey_kyber512_keygen(EVP_PKEY_CTX *ctx, EVP_PKEY *pkey) {
   return 1;
 }
 
+static int pkey_kyber512_encapsulate(EVP_PKEY_CTX *ctx,
+                                     uint8_t *ciphertext,
```

Comment:
Okay, the typical style is not to add newlines for every parameter. But it's not like I'm going to complain a lot about it. Just adding it as a note.